### PR TITLE
Feat: New landing page

### LINKED ISSUE #220: CNCFSD-2166 SOPS Website
https://cncfservicedesk.atlassian.net/jira/servicedesk/projects/CNCFSD/queues/custom/1/CNCFSD-2166

### PR DESCRIPTION
**Pull request by thisisobate**
Fixes https://github.com/cncf/techdocs/issues/220

### CODE REVIEW

**Review by nate-double-u — APPROVED** [review]
Thanks for this @thisisobate.

/cc @hiddeco @sabre1041

File: hugo.toml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 baseURL = "/"
-title = "SOPS: Secrets OPerationS"
+title = "SOPS: Secrets Operations"
```
Comment (nate-double-u) [inline hugo.toml]: I think they intentionally capitalize it this way.

**Review by nate-double-u — CHANGES_REQUESTED** [review]
Looking at the docs page, it looks like the dark mode has made some of the text unreadable.

https://deploy-preview-6--getsops.netlify.app/docs/#stable-release
https://getsops.netlify.app/docs/#stable-release

---

IMG_0169

IMG_0168

File: hugo.toml (hunk 1 above)
Comment (thisisobate) [reply hugo.toml]: It looks like a typo to me though. Maybe it's best we confirm from the maintainers themselves?
@hiddeco @sabre1041 can you please confirm the appropriate title?

Comment (thisisobate) [reply hugo.toml]: Oh...just figured this was done on purpose.

**Review by nate-double-u — APPROVED** [review]
Thanks for the updates @thisisobate!

/approve

**Comment by nate-double-u** [comment]
Merging as a completed base site.

**Pull request merged**